### PR TITLE
fix: fix FTX configs

### PR DESCRIPTION
**Pull request by ycdesu**
In order to support FTX in config, we have to add FTX to the enabled exchanges list. I also modify the readme and default config path. According to `.gitignore`, the default config path should be `config/bbgo.yaml`. Then user can follow the readme to run bbgo without specifying `--config` explicitly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# bbgo
+bbgo
```
Comment (c9s) [inline README.md:1]: I think this is the title?

File: README.md [hunk 2]
```diff
@@ -67,13 +67,19 @@ MAX_API_SECRET=
 FTX_API_KEY=
 FTX_API_SECRET=
 # specify it if credentials are for subaccount
-FTX_SUBACCOUNT_NAME=
+FTX_SUBACCOUNT=
```
Comment (c9s) [inline README.md:70]: thanks~

Comment (c9s) [reply README.md:70]: oh it should be `FTX_API_SUBACCOUNT`

File: pkg/cmd/root.go [hunk 3]
```diff
@@ -54,7 +54,7 @@ var RootCmd = &cobra.Command{
 
 func init() {
 	RootCmd.PersistentFlags().Bool("debug", false, "debug flag")
-	RootCmd.PersistentFlags().String("config", "bbgo.yaml", "config file")
+	RootCmd.PersistentFlags().String("config", "config/bbgo.yaml", "config file")
```
Comment (c9s) [inline pkg/cmd/root.go:57]: would prefer `bbgo.yaml` it simplifies the installation :-)

File: README.md (hunk 2 above)
Comment (ycdesu) [reply README.md:70]: It should be `FTX_SUBACCOUNT` actually https://github.com/c9s/bbgo/blob/main/pkg/cmd/cmdutil/exchange.go#L51

Comment (c9s) [reply README.md:70]: oh you are right

Comment (c9s) [reply README.md:70]: do you think we should the API to the prefix?

**Review by c9s — APPROVED** [review]

**Pull request merged**